_tests/travis-checks.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+# Execute Markdown Linter
+mdl README.md _posts _pages --style _linter/markdown-linter-style.rb
+
+# Github Pages needs the baseurl field to be "m-lab.github.io" to render properly in forked 
+# repos, but when we deploy to production, we need the baseurl field to be empty. This 
+# command replaces "m-lab.github.io" with an empty string.
+sed -e '/^baseurl/s/\/m-lab\.github\.io//' _config.yml > _config-deploy.yml
+
+# Run HTMLProofer
+bundle exec jekyll build --config _config-deploy.yml
+bundle exec htmlproofer ./_site --only-4xx --check-html --file-ignore "/mlab_observatory/","/ndt/" $1
```

Comment:
OK, I like this suggestion.  Made the argument ```--quick``` now and then in the travis-checks file it looks for the first argument and replaces "quick" with "disable-external" if found.  If it doesn't find "quick" then it will not execute HTMLProofer with any other flags.  Could also do this with a more traditional if statement I suppose, but seemed like this implementation was a little more succinct.  If you had something else in mind we can change that piece.

For now marking this as fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/3f6000223f6819c18a5133a60e77369a825dff00